### ISSUE TITLE
Node_Graph example crashes

### ISSUE THREAD

**Issue by shujaatak** [apertus-open-source-cinema/narui#26]
node_graph example crashes:
```
thread 'main' panicked at 'Failed to acquire next image: Timeout', narui_core/src/vulkano_render/render.rs:196:31
```

**Comment by shujaatak** [comment]
slider example crashes whenever I quickly move the slider's handle otherwise it doesn't crash. Similarly node_graph example crashes whenever I quickly move the mouse to join nodes.

**Comment by rroohhh** [comment]
Can you provide some more information on your system?
What OS are you using?
What GPU are you using?
What GPU driver?
Can upload the output of `vulkaninfo` or a equivalent somewhere?

**Comment by anuejn** [comment]
I was able to reproduce this on my system and pushed a fix in [ea08325](https://github.com/apertus-open-source-cinema/narui/commit/ea083257f3a098c5d5786eefb779df5875bc3a33)
